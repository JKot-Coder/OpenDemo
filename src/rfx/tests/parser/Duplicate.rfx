// :PARSER

Foo: {
    Duplicate: "?"
},

Bar: {
    << Foo,
    Duplicate: "no"
},

Test: {
    Duplicate: false,
    Duplicate: -1,
}