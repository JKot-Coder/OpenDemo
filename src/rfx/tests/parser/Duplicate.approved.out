Diagnostic:[
    ../src/rfx/tests/parser/Duplicate.rfx(13): error 20014: Duplication of the key 'Duplicate'
        Duplicate: -1,
        ^~~~~~~~~
]
